src/OSPSuite.UI/Views/Charts/AxisSettingsView.cs

```diff
@@ -91,10 +93,10 @@ public override void InitializeBinding()
          createColumn(x => x.GridLines, AxisOptionsColumns.GridLines, _gridLinesRepository);
 
          createColumn(x => x.DefaultColor, AxisOptionsColumns.DefaultColor, _colorRepository, toolTip: ToolTips.DefaultCurveColor)
-            .WithEditorConfiguration(disableForXAxis);
```

Comment:
Instead of having the repository with editor that cannot be clicked, I am not showing any editor